src/Module.zig

```diff
@@ -80,6 +80,9 @@ deletion_set: ArrayListUnmanaged(*Decl) = .{},
 /// Error tags and their values, tag names are duped with mod.gpa.
 global_error_set: std.StringHashMapUnmanaged(u16) = .{},
 
+/// error u16 -> []const u8 for fast lookups for @intToError at comptime
+inv_error_map: ArrayListUnmanaged([]const u8) = .{},
```

Comment:
name suggestion:

```suggestion
error_name_list: ArrayListUnmanaged([]const u8) = .{},
```

they're both inverted from the other one's perspective - I think this name communicates better what it is